pkg/parcacol/normalizer.go

```diff
@@ -320,8 +337,18 @@ func (n *MetastoreNormalizer) NormalizeLocations(
 		mappingId := ""
 		if location.MappingId != 0 {
 			mappingIndex := location.MappingId - 1
-			mappingNormalizationInfo := mappings[mappingIndex]
-
+			mappingNormalizationInfo := mappingsInfo[mappingIndex]
+
+			if normalizeWithBaseAddress {
+				m := mappings[mappingIndex]
+				addr, err = NormalizeAddress(addr, executableInfo[mappingIndex], m.MemoryStart, m.MemoryLimit, m.FileOffset)
+				if err != nil {
+					// This should never happen, since we already checked that
```

Comment:
Maybe we should at least add a metric, WDYT?

Reply:
makes sense!